apps/src/applab/errorHandler.js

```diff
@@ -16,7 +16,7 @@ function outputApplabConsole(output) {
     if (debugOutput.textContent.length > 0) {
       debugOutput.textContent += '\n' + output;
     } else {
-      debugOutput.textContent = output;
+      debugOutput.textContent = '' + output;
```

Comment:
I understand this is a string coercion but it looks strange enough that it might be worth a comment.  Or is there a reason you didn't just use `String(x)`? [It seems like it does what you want](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#String_conversion).
```
debugOutput.textContent = String(output);
```